### PR TITLE
PR for our new dataset and publication release!

### PR DESCRIPTION
**Pull request by etienne87**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,20 +46,41 @@ Now you can start by running a baseline either by looking into [the last results
 
 ## Evaluation using the COCO API
 
+
+### DISCLAIMER: New Dataset! 
+
+To account for the new 1 Megapixel Dataset described in our recently accepted NeurIPS submission: "Learning to detect 1 Megapixel Event Camera", the format has slightly changed.
```
Comment (asironi) [inline README.md:52]: paper title is "Learning to Detect Objects with a 1 Megapixel Event Camera"

Comment (etienne87) [reply README.md:52]: ah thanks yes will change that

File: src/io/box_filtering.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    """Filters boxes according to the paper rule. 
+
+    Args:
+        boxes (np.ndarray): structured box array with fields "ts" or "t"
+    """
+    ts = boxes['ts'] if 'ts' in boxes.dtype.names ele boxes['t']
+    width = boxes['w']
+    height = boxes['h']
+    diag = width**2+height**2
+    skip_ts = int(1e5)
```
Comment (davidenitti) [inline src/io/box_filtering.py:25]: skip_ts is 0.5s? here it's 0.1 I think

File: src/io/box_filtering.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    mask = (ts>ts[0]+skip_ts)*(diag >= min_box_diag**2)*(width >= min_box_side)*(height >= min_box_side)
```
Comment (davidenitti) [inline src/io/box_filtering.py:26]: is ts[0] the first event time?

File: src/io/box_filtering.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+def filter_boxes(boxes, skip_ts=int(1e5), min_box_diag=30, min_box_side=10):
+    """Filters boxes according to the paper rule. 
+
+    Args:
+        boxes (np.ndarray): structured box array with fields "ts" or "t"
+    """
+    ts = boxes['ts'] if 'ts' in boxes.dtype.names ele boxes['t']
+    width = boxes['w']
+    height = boxes['h']
+    diag = width**2+height**2
```
Comment (davidenitti) [inline src/io/box_filtering.py:24]: refactor diag in diag_squared or something like that

File: src/io/box_filtering.py [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+1. skip all boxes before 0.5s (before we assume it is unlikely you have sufficient historic)
+2. filter all boxes whose diagonal <= min_box_diag**2 and whose side <= min_box_side
+
+Copyright: (c) 2019-2020 Prophesee
+"""
+from __future__ import print_function
+import numpy as np
+
+
+def filter_boxes(boxes, skip_ts=int(1e5), min_box_diag=30, min_box_side=10):
```
Comment (davidenitti) [inline src/io/box_filtering.py:15]: is this at the original scale or downscaled? we maybe need to clarify this

File: src/io/box_filtering.py (hunk 2 above)
Comment (etienne87) [reply src/io/box_filtering.py:25]: ah correct thanks

File: src/io/box_filtering.py (hunk 3 above)
Comment (etienne87) [reply src/io/box_filtering.py:26]: yes

File: src/io/box_filtering.py (hunk 4 above)
Comment (etienne87) [reply src/io/box_filtering.py:24]: ok

File: src/io/box_filtering.py (hunk 5 above)
Comment (etienne87) [reply src/io/box_filtering.py:15]: right, i think we use min_box_diag=30 in our case, both for imerit and the moorea dataset (60/2 and 20/2). We have not evaluated on Full resolution

File: src/io/box_filtering.py (hunk 3 above)
Comment (davidenitti) [reply src/io/box_filtering.py:26]: is it possible to set the initial time? if we don't have events basically we skip more.

File: src/io/box_filtering.py (hunk 5 above)
Comment (etienne87) [reply src/io/box_filtering.py:15]: will add a bit of text in the README

File: src/io/box_filtering.py (hunk 3 above)
Comment (etienne87) [reply src/io/box_filtering.py:26]: right, i think we should assume that initial_time is zero no?

**Pull request merged**